agent/agents/process/process_test.go

```diff
@@ -189,3 +189,30 @@ func TestProcess(t *testing.T) {
 		require.EqualError(t, err, "os: process already finished", "child process with pid %v is not killed", pid)
 	})
 }
+
+func TestMatchLogLevel(t *testing.T) {
+	tests := []struct {
+		testName string
+		line     string
+		expected logrus.Level
+	}{
+		{"info", `ts=2022-06-14T21:43:42.984Z caller=mysqld_exporter.go:492 level=info msg="Starting mysqld_exporter"`, logrus.InfoLevel},
+		{"warn", `ts=2022-06-14T21:43:42.984Z caller=mysqld_exporter.go:492 level=warn msg="Starting mysqld_exporter"`, logrus.WarnLevel},
+		{"error", `ts=2022-06-14T21:43:42.984Z caller=mysqld_exporter.go:492 level=error msg="Starting mysqld_exporter"`, logrus.ErrorLevel},
+		{"panic to error", `ts=2022-06-14T21:43:42.984Z caller=mysqld_exporter.go:492 level=panic msg="Starting mysqld_exporter"`, logrus.ErrorLevel},
+		{"fatal to error", `ts=2022-06-14T21:43:42.984Z caller=mysqld_exporter.go:492 level=fatal msg="Starting mysqld_exporter"`, logrus.ErrorLevel},
```

Comment:
If method extract log level, it should return `panic` if it is `panic`. It must not amend this value.
If needed, extra logic should be added to substitute level. But it should not be a part of extraction.

Reply:
@ritbl done, can you review?